lighthouse-cli/run.js

```diff
@@ -44,7 +44,7 @@ function parseChromeFlags(flags = '') {
       // i.e. `child_process.execFile("lighthouse", ["http://google.com", "--chrome-flags='--headless --no-sandbox'")`
       // the following regular expression removes those wrapping quotes:
       .map((flagsGroup) => flagsGroup.replace(/^\s*('|")(.+)\1\s*$/, '$2').trim())
-      .join(' ');
+      .join(' ').trim();
```

Comment:
Is this change needed?

Reply:
yup it is thanks to the bin.js change (that i wrote).

`cliFlags.chromeFlags` was `=== [undefined, '--enable-features=blahblah']`  which was then joined with ' ' and then that caused some problems in the yargs parser. 

there's other solutions, but doing a trim here seemed like the safest move.